electron-app/src/components/inputs/AssetSelect.vue

```diff
@@ -48,6 +49,15 @@ const { mapState } = createNamespacedHelpers('balances');
 export default class AssetSelect extends Vue {
   supportedAssets!: SupportedAsset[];
 
+  @Prop({ required: false, default: null })
+  items?: string[];
+
+  @Prop({ required: false, default: '' })
+  hint?: string;
```

Comment:
```suggestion
  hint!: string;
```
This also will never be undefined because of the default value.

Reply:
Got it.